### DISCUSSION TITLE
Create a verifier on mainnet - Still in processing state over 5days

### DISCUSSION THREAD

**Discussion by huyta** [Web3Auth/Web3Auth#797] [Help]
Hello Folks,

I just create a verifier on mainnet (**op3n-firebase-verifier**). It's still in Processing state over 5days

Do you have any idea about this? (I also created a verifier on testnet an cyan, just take about 30mins and the verifiers are live)

Thanks,
Huy

**Comment by shahbaz17** [comment]
This is fixed now.

**Reply by huyta** [reply]
Hello team,

Any update on this?

Thanks,
Huy

**Reply by BboyStatix** [reply]
@huyta Apologies! We're currently looking into it. Sorry for the delay this isn't normally the case.

**Reply by BboyStatix** [reply]
@huyta could you check again?

**Reply by huyta** [reply]
Thank you @BboyStatix. It's live now!
